### DISCUSSION TITLE
Bonsai x DeepLabCut not working: multianimal pose estimation not possible?

### DISCUSSION THREAD

**Discussion by NunoMac** [bonsai-rx/bonsai#1245] [Q&A] [answered]
Hi, I have trained a model for multiple animals with DLC and it is working, and exported the model to the .pb file with the `deeplabcut.export_model(config_path)` .

When running just FileCapture with PredictPose, I get an error:
Input string was not in a correct format.

I am wondering: is it not possible to get the pose of 2 animals at the same time in Bonsai?
[Here](https://github.com/DeepLabCut/DeepLabCut/blob/main/docs/HelperFunctions.md) I actually read this:
_allows you to export a well-trained single animal model_
but would like to confirm if that is actually a limitation...

Thanks!

**Comment by bruno-f-cruz** [comment, marked as answer]
Can you make sure the regional format of your machine is compatible with "." instead of "," as a decimal separator? American format should work.

**Reply by NunoMac** [reply]
That was exactly the problem! Would never get there, thanks!

(now that it is working, it also crashes in a few seconds, but if I can't solve it, maybe I'll have to open another topic....)
